docs/DatabaseCatalogAPI_IAP.yaml

```diff
@@ -0,0 +1,792 @@
+openapi: 3.0.0
```

Comment:
Having two files that are identical except for one line hurts. There's no way to create a copy and replace the line with a sed?

Reply:
My only concern is how clear it is to have a script that modifies the schema on the fly. At the same time, it's indeed painful to have two files with 99.9 % of matching lines. I created an issue to address this after MVP, #146.